composer.json

```diff
@@ -0,0 +1,10 @@
+{
+    "require-dev": {
+        "squizlabs/php_codesniffer": "^3.7"
+    },
+    "autoload": {
+        "psr-4": {
+            "App\\": "app/"
+        }
+    }
+}
```

Comment:
This is ok, just don't forget to add some information for the package (name, description, visibility, etc)

Reply:
Done